src/content/developers/tutorials/yellow-paper-evm/index.md

```diff
@@ -0,0 +1,396 @@
+---
+title: Understanding the Yellow Paper's EVM Specifications
+description: Understanding the part of the Yellow Paper, the formal specifications for Ethereum, that explains the ethereum virtual machine (EVM).
+author: "qbzzt"
+tags: ["evm", "yellow paper", "specifications"]
+skill: intermediate
+lang: en
+sidebar: true
+published: 2022-05-15
+---
+
+[The Yellow Paper](https://ethereum.github.io/yellowpaper/paper.pdf) is the formal specifications for Ethereum. 
```

Comment:
> The Yellow Paper is the formal specifications for Ethereum

Reads a bit weird IMO - should "specifications" be plural? 

I'd suggest either:

> The Yellow Paper is the formal specification for Ethereum

Or 

> The Yellow Paper contains the formal specifications for Ethereum